prusti-viper/src/encoder/mir/generics/interface.rs

```diff
@@ -113,6 +113,6 @@ impl<'v, 'tcx: 'v> MirGenericsEncoderInterface<'tcx> for super::super::super::En
     }
     fn encode_param(&self, name: Symbol, index: u32) -> vir_high::ty::TypeVar {
         let identifier = format!("{}${}", name.as_str(), index);
-        vir_high::ty::TypeVar::new(identifier)
+        vir_high::ty::TypeVar::GenericType(vir_high::ty::GenericType { name: identifier })
```

Comment:
```suggestion
        vir_high::ty::TypeVar::generic_type(identifier)
```

Reply:
That doesn't seem to work/compile. Should `generic_type` be generated?

Reply:
Try adding `#[derive_helpers]` on the `TypeVar` definition.

Reply:
thanks, done